site/playwright.config.ts

```diff
@@ -1,6 +1,8 @@
 import type { PlaywrightTestConfig } from "@playwright/test";
 import { devices } from "@playwright/test";
 
+(await import("dotenv-flow")).config();
```

Comment:
Makes sure Playwright tests are aware of `.env.local` (used for `NEXT_PUBLIC_NPM_PUBLISHING`)